VimeoUpload/Descriptor System/DescriptorManagerDelegate.swift

```diff
@@ -30,21 +30,21 @@ import Foundation
 
 @objc public protocol DescriptorManagerDelegate: class
 {
-    optional func didLoadDescriptors(descriptors descriptors: Set<Descriptor>)
-    optional func didSaveDescriptors(count count: Int)
-    optional func didFailToLoadDescriptor(error error: NSError)
+    @objc optional func didLoadDescriptors(descriptors: Set<Descriptor>)
```

Comment:
I believe we need to. Optional protocol methods only exist in Objective-C but not in Swift.